### ISSUE TITLE
Scrape list of Skills

### ISSUE THREAD

**Issue by tedvanderveen** [joeyism/linkedin_scraper#27]
In the latest version of the LinkedIn website, there does not seems te be a directory of Skills anymore. Could this be added to this library in any way?

**Comment by GoswamiTushar** [comment]
Hello, I would to work on this enhancement, I'm new and just learnt web scraping.

**Comment by arnav-deep** [comment]
@joeyism 
I can add the feature to retrieve the skill by tomorrow. If no one is working on it, assign it to me.
I have gone through code and can write it in the same manner perfectly.

**Comment by priyankagarg112** [comment]
Is this feature added?

**Comment by arnav-deep** [comment]
> Is this feature added?

I don't think so. I've been looking for the go ahead. Haven't gotten any. I guess I'll try doing it on my own.

**Comment by joeyism** [comment]
@priyankagarg112 @arnav-deep If you have a fix and want to submit a merge request, I'll gladly take a look at it

**Comment by RubenPants** [comment]
Any update on this?

**Comment by sametsoekel** [comment]
agree with @RubenPants , is this feature added ? I guess none of enthusiastic people here had no time or forgot to add that cuz not included in latest version. @joeyism I willingly could do this if no one is workin' on it !

**Comment by joeyism** [comment]
@sametsoekel if you can, that would be great

**Comment by albxr** [comment]
updates?

**Comment by financeirotalenses** [comment]
updates?
